Include/internal/pycore_interp.h

```diff
@@ -138,6 +138,8 @@ struct _is {
            or the size specified by the THREAD_STACK_SIZE macro. */
         /* Used in Python/thread.c. */
         size_t stacksize;
+        /* The mutex lock of the current executing thread. */
```

Comment:
```suggestion
        /* The lock that protects this struct. */
```

...or drop the comment and move the field to the top of the struct, like we do with `_PyRuntimeState`.